src/Nethermind/Nethermind.Merge.Plugin/InvalidChainTracker/InvalidBlockInterceptor.cs

```diff
@@ -97,4 +98,14 @@ private static bool ShouldNotTrackInvalidation(BlockHeader header)
     {
         return !HeaderValidator.ValidateHash(header);
     }
+
+    private static bool ShouldNotTrackInvalidation(Block block)
+    {
+        if (ShouldNotTrackInvalidation(block.Header)) return true;
+
+        // Body does not match header, but it does not mean the hash that the header point to is invalid.
+        if (!BlockValidator.ValidateTxRootMatchesTxs(block, out Keccak ignored)) return true;
```

Comment:
I think convention on ignored out values is to use `_`

Reply:
Done